### PR TITLE
[#16422] Merge profile and profile overview into one map in status-go

### LINKED ISSUE #16422: Merge profile and profile overview into one map in status-go
Currently, its done in the subscriptions, we merge profile overview and profile because some settings (for example customization color) we have only in the profile overview map, it should be done in status-go

related issue: https://github.com/status-im/status-go/issues/3684

### LINKED ISSUE #16444: Selected accent color is not applied inside newly created account until re-login
**Steps:** 

1. Proceed with creating a new user
2. Select some non-default accent color on Profile creation screen
3. Pay attention if selected accent color has been applied inside account

**Actual result:** selected accent color is not applied until re-login

https://github.com/status-im/status-mobile/assets/97245802/b3910156-b931-49e7-87e1-c95b3ddc5b1a

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes https://github.com/status-im/status-mobile/issues/16444
fixes https://github.com/status-im/status-mobile/issues/16422

### CODE REVIEW

File: src/status_im/browser/core.cljs [hunk 1]
```diff
@@ -619,3 +619,40 @@
                                   bookmarks)
```
Comment (flexsurfer) [inline src/status_im/browser/core.cljs:619]: just moved, no changes

File: src/status_im/group_chats/core.cljs [hunk 2]
```diff
@@ -9,7 +9,8 @@
             [status-im2.constants :as constants]
```
Comment (flexsurfer) [inline src/status_im/group_chats/core.cljs:9]: just moved, no changes

File: src/status_im/utils/logging/core.cljs [hunk 3]
```diff
@@ -26,6 +26,17 @@
     (string/join "\n" (log/get-logs-queue))
```
Comment (flexsurfer) [inline src/status_im/utils/logging/core.cljs:26]: just moved, no changes

File: src/status_im/wallet/core.cljs [hunk 4]
```diff
@@ -21,7 +21,6 @@
     [status-im.utils.core :as utils.core]
```
Comment (flexsurfer) [inline src/status_im/wallet/core.cljs:21]: just moved, no changes

File: src/status_im2/contexts/chat/events.cljs [hunk 5]
```diff
@@ -17,7 +17,10 @@
             [status-im.utils.types :as types]
```
Comment (flexsurfer) [inline src/status_im2/contexts/chat/events.cljs:17]: just moved, no changes

File: src/status_im2/contexts/profile/config.cljs [hunk 6]
```diff
@@ -1,7 +1,9 @@
 (ns status-im2.contexts.profile.config
```
Comment (flexsurfer) [inline src/status_im2/contexts/profile/config.cljs:1]: just moved, no changes

**Review by Parveshdhull — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]
nice!

**Review by smohamedjavid — APPROVED** [review]
🚀

**Comment by pavloburykh** [comment]
@flexsurfer thanx for the PR. no issues from my side. Ready for merge.

**Pull request merged**